modules/ensemble/lib/widget/input/form_time.dart

```diff
@@ -42,33 +46,53 @@ class Time extends StatefulWidget
   Map<String, Function> setters() {
     var setters = _controller.textPlaceholderSetters;
     setters.addAll({
-      'initialValue': (value) =>
-          _controller.initialValue = Utils.getTimeOfDay(value),
+      'initialValue': (value) {
+        _controller.initialValue = Utils.getTimeOfDay(value);
+        _controller.value =
+            _controller.initialValue; // Set the current value to initialValue
+      },
       'onChange': (definition) => _controller.onChange =
-          EnsembleAction.from(definition, initiator: this)
+          EnsembleAction.from(definition, initiator: this),
+      'useIOSStyleTimePicker': (value) =>
+          _controller.useIOSStyleTimePicker = value,
+      'use24hFormat': (value) => _controller.use24hFormat = value,
     });
     return setters;
   }
 }
 
 class TimeController extends FormFieldController with HasTextPlaceholder {
   TimeOfDay? value;
-
+  TimeOfDay? initialValue;
+  EnsembleAction? onChange;
+  bool useIOSStyleTimePicker = Platform.isIOS;
+  bool use24hFormat = false;
+  
   Text prettyValue(BuildContext context) {
-    if (value != null) {
-      return Text(value!.format(context),
-          style: TextStyle(fontSize: fontSize?.toDouble()));
-    } else {
-      return Text(placeholder ?? 'Select a time', style: placeholderStyle);
+    if (value == null) {
+      return Text(
+        placeholder ?? 'Select a time',
```

Comment:
is there a localized default placeholder that Flutter has?

Reply:
yup, using it from `MaterialLocalizations `.